apps/condo/domains/organization/access/Organization.js

```diff
@@ -87,6 +88,9 @@ async function canManageOrganizations ({ authentication: { item: user }, operati
     const hasDirectAccess = await canDirectlyManageSchemaObjects(user, listKey)
     if (hasDirectAccess) return true
 
+    const hasDirectAccessToFieldUpdating = await canDirectlyManageSchemaFields(user, listKey, originalInput, operation)
```

Comment:
Maybe we should merge this with `canDirectlyManageSchemaObjects`?

Reply:
Will it be so obvious that function `canDirectlyManageSchemaObjects` checks not only direct access to entire schema, but also direct access only to some of its fields?